### DISCUSSION TITLE
Thermal conductivity from BTE

### DISCUSSION THREAD

**Discussion by brucefan1983** [brucefan1983/GPUMD#696] [General]
Calculating the lattice thermal conductivity based on the Boltzmann transport equation (BTE) and lattice dynamics (harmonic and anharmonic).

**Comment by brucefan1983** [comment]
There is already a fast GPU-based BTE code called `GPU_PBTE`: https://gitlab.com/xiaokun.gu/GPU_PBTE

I think there is no need for `GPUMD` to achieve similar goals any more.

**Comment by shdchen** [comment]
Perhaps, it would still be helpful to have another independent GPU implementation of the BTE code. Different implementation may have different efficiency. And it would be nice to couple the BTE code with NEP. NEP-BTE should be able to achieve the accuracy of DFT-BTE, with much less computational cost.

**Comment by shdchen** [comment]
Look forward to NEP-BTE.   :-)

**Comment by brucefan1983** [comment]
OK, but this will have very low priority!

**Comment by shdchen** [comment]
OK. Understood. Thanks.
